cmd/graphql/resolvers/user.resolver.go

```diff
@@ -0,0 +1,113 @@
+package resolvers
```

Comment:
Ah, I did think of that, but then again I would have many user.go files... which makes it a bit hard to track which user go file I am in while in development. If it's recommended, i will change it on the releases (master branch)

Reply:
The reason why I say this is because every package will need the name `something.resolver` if you go down this route. It just leads to boilerplate which is non-descriptive in your resolvers package. Ultimately, not a huge deal though.

At the very least, you should use `_` instead of `.` if you have a multipart name.